xdcs-agent/src/main/python/xdcs/agent_cli.py

```diff
@@ -1,2 +1,15 @@
+import os
+
+_config_locations = [
```

Comment:
tego chyba tu nie powinno być

Reply:
bez różnicy raczej bo i tak to nie jest teraz używane a pomaga w sprawdzeniu tego pipa, ale moge wywalić

Reply:
to jest copy-paste